### ISSUE TITLE
-quiet killing all output for 'find' verb

### ISSUE THREAD

**Issue by rsdoiel** [caltechlibrary/dataset#46]
The following returns empty output, should output a list of ideas.

```
   bin/dataset -quiet -ids-only -size=100000 -sort="-sort_date,+title" find New-CaltechAUTHORS.bleve 'GALCIT'
```

```
   bin/dataset -ids-only -size=100000 -sort="-sort_date,+title" find New-CaltechAUTHORS.bleve 'GALCIT'
```

Returns ids, -quiet should only suppress errors for 'find'.

**Comment by rsdoiel** [comment]
if there any reason to have -quiet supress output other than messages like "OK"?  Should -ids-only set -quiet to false?

**Comment by rsdoiel** [comment]
Fixed for v0.041 release
